developer/reference/runtime/register-component.mdx

```diff
@@ -146,6 +149,67 @@ runtime.registerComponent(Cube, {
 });
 ```
 
+### Adding a description
+
+This example shows how to add an `Circle` component with a rich description. We
+use markdown formatting for our tooltip descriptions.
+
+In the builder component tray and panel, we can see an info icon next to the component label.
+
+<Frame>
+  <img
+    src="/images/register-custom-desc1.png"
+    alt="custom component info icon"
+  />
+</Frame>
+
+Hover over the label to open the description tooltip.
+
+<Frame>
+  <img
```

Comment:
Can we make this bigger and also put this in a more specific folder. I would suggest `/images/developer/reference/register-component/`.

Reply:
I think we should also move the image to come after the code snippet. That way, you can see the result of the code snippet